src/test/java/com/wikia/webdriver/common/logging/VelocityWrapper.java

```diff
@@ -0,0 +1,196 @@
+package com.wikia.webdriver.common.logging;
+
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+
+import java.io.StringWriter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.testng.internal.Utils.escapeHtml;
+
+
+public class VelocityWrapper {
+
+  private static final String LAST_LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/lastLogRow.vm";
+  private static final String LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRow.vm";
+  private static final String LOG_ROW_WITH_LINK_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRowWithLink.vm";
+  private static final String IMAGE_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/image.vm";
+  private static final String ERROR_LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/errorLogRow.vm";
+  private static final String LOG_ROW_WITH_SCREENSHOT_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRowWithScreenshot.vm";
+  private static final String LINK_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/link.vm";
+  private static final String FIRST_LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/firstLogRow.vm";
+  private static final String BUTTON_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/button.vm";
+  private static final String HEADER_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/header.vm";
+
+  public VelocityWrapper(){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Hide this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)